File: arrays/q3.c
#include<stdio.h>

void main(){
    int n,sum;
    printf("Enter number of values to enter n : ");
    scanf("%d", &n);   
    int arr[n];
    // take input 
    for(int i = 0 ; i < n ; i++){
       printf("Enter value at index %d\n",i); 
       scanf("%d", &arr[i]);
       sum += arr[i];
    }
    
    printf("Sum is %d\n",sum);

}

File: as3/control/q3.py
import math
n = int(input("Enter a number : "))

# Check if the number is a perfect number 

sum = 0 
for i in range(1,math.ceil(n/2)+1):
    if n % i == 0 : 
        sum += i;

if sum == n : 
    print("PERFECT NUMBER")
else : 
    print("NOT A PERFECT NUMBER")